### DISCUSSION TITLE
I unlocked 1 badge, when can I find the rest

### DISCUSSION THREAD

**Discussion by ItzRealMe** [community/community#58074] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

So, I unlocked 1 badge on my profile, does anyone know where I can look to find information about getting the rest?

**Comment by Thinkright20** [comment, marked as answer]
Hi, I'm pretty sure you can see information on the other badges at this repo:
https://github.com/Schweinepriester/github-profile-achievements

Its very organized and has good info for what ur looking for.

**Reply by ItzRealMe** [reply]
thanks so much again

**Reply by LuanSchneider** [reply]
response #1

**Reply by nubelaeye** [reply]
Please select as answer.

**Reply by inzayn99** [reply]
response #75

**Reply by AbdullahAssi** [reply]
for badge

**Reply by Jakhaishya** [reply]
thank you so much

**Reply by DanielBrisch** [reply]
Response #1

**Reply by Kentzyann** [reply]
Kejeje

**Reply by B-azeddine** [reply]
are we always need someone to mark our answers  to get that badge ?

**Reply by Arslan3x5** [reply]
Thanks

**Reply by Ankitjarwall** [reply]
hii thank for the youtube video, i just comment two. please mark my response as answer. Thank you

**Reply by brendondev** [reply]
Thank you.

**Reply by HemantMedhsia** [reply]
kindly accept it for now.

**Comment by paras-017** [comment]
https://github.com/github-profile-achievements/english

**Reply by LuanSchneider** [reply]
response #2

**Reply by GHOST-ACCAUNT** [reply]
Hello everyone!!!
I'm Kamronbek
I'm UZBEK
Hullas bolar men Uzbekman karilamanglar basharalaringni buzaman lekin

**Reply by AbdullahAssi** [reply]
response#2

**Reply by ANIRUDHSINH01** [reply]
Thanks

**Reply by Kentzyann** [reply]
Shhshs

**Reply by Arslan3x5** [reply]
Thank

**Reply by yasiramri** [reply]
Thanks

**Reply by ghost** [reply]
I will create 8 Q&A discussions in my repository. If interested, up to 4 people can answer these questions. Once I select their answers as accepted, those 4 individuals will receive the badge. If you want to earn the badge, let me know, and I'll create the 4 Q&A discussions.

**Reply by luismede** [reply]
> I will create 8 Q&A discussions in my repository. If interested, up to 4 people can answer these questions. Once I select their answers as accepted, those 4 individuals will receive the badge. If you want to earn the badge, let me know, and I'll create the 4 Q&A discussions.

Hi, I want 👊

**Reply by HemantMedhsia** [reply]
yes, I want that please create this.

**Comment by nubelaeye** [comment]
Can anyone tell how many badges all ?
😯

**Comment by nubelaeye** [comment]
Shouldn't you think that github should also build a badge section and the a badges for the streak on commits ?

**Comment by Faiaz3k** [comment]
use this https://github.com/Schweinepriester/github-profile-achievements.git

**Comment by ANIRUDHSINH01** [comment]
Thanks

**Comment by Ankitjarwall** [comment]
hii thank for the youtube video, i just comment two. please mark my response as answer. Thank you

**Comment by luismede** [comment]
Hi
Use this: https://github.com/github-profile-achievements/english
will help you

**Comment by ghost** [comment]
heheheh

**Comment by Gabriela164** [comment]
Hi
